managereport.php

```diff
@@ -164,8 +164,10 @@
         if (!empty($export)) {
             foreach ($export as $e) {
                 if ($e) {
+                    $cid = $context->id;
+                    $url = moodle_url::make_pluginfile_url($cid, 'block_configurable_reports', 'export', null, '/', $e);
                     $download .= '<a href="viewreport.php?id='.$r->id.'&amp;download=1&amp;format='.$e.'">'.
-                        '<img src="'.$CFG->wwwroot.'/blocks/configurable_reports/export/'.$e.'/pix.gif" alt="'.$e.'">'.
+                        '<img src="'.$url.'" alt="'.$e.'">'.
```

Comment:
please convert this to html_writer

Reply:
Hey @brendanheywood - this has been fixed in another pr - https://github.com/jleyva/moodle-block_configurablereports/pull/204 (i couldn't push anything to here). Please review.

Thanks,
Anu